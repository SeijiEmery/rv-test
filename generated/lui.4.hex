0000000    b7  a5  51  ef  13  c6  85  46  73  00  10  00  13  00  00  00
0000020    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    73  00  10  00                                                
0000044
