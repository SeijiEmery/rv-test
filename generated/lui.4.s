lui a1, 0xEF51A
    xori a2, a1, 0x468
    ebreak
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
