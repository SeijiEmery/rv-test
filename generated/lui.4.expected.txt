tests/lui.test.s:30
pc = 80
R11 = 18446744073429688320
R12 = 18446744073429689448
